qa/L0_perf_analyzer_capi/test.sh

```diff
@@ -42,7 +42,7 @@ fi
 export CUDA_VISIBLE_DEVICES=0
 
 CLIENT_LOG="./perf_analyzer.log"
-PERF_ANALYZER=../clients/perf_analyzer
+PERF_ANALYZER=/opt/tritonserver/bin/perf_analyzer
```

Comment:
is this intended? will it use the most recently built PA during CI?

Reply:
Good catch! Fixed.